examples/demo.rs

```diff
@@ -135,6 +135,29 @@ impl mq::EventHandler for Stage {
     fn key_up_event(&mut self, _ctx: &mut mq::Context, keycode: mq::KeyCode, keymods: mq::KeyMods) {
         self.egui_mq.key_up_event(keycode, keymods);
     }
+
+    fn touch_event(
+        &mut self,
+        ctx: &mut mq::Context,
+        phase: mq::TouchPhase,
+        id: u64,
+        x: f32,
+        y: f32,
+    ) {
+        if phase == mq::TouchPhase::Started {
+            self.mouse_button_down_event(ctx, mq::MouseButton::Left, x, y);
+        }
+
+        if phase == mq::TouchPhase::Ended {
+            self.mouse_button_up_event(ctx, mq::MouseButton::Left, x, y);
+        }
+
+        if phase == mq::TouchPhase::Moved {
+            self.mouse_motion_event(ctx, x, y);
+        }
```

Comment:
This could be moved into `EguiMq::touch_event` to lessen the amount of code the user has to write

Reply:
could also be a `match phase`